### PR TITLE
[OSDOCS-5027] Add RN clarifying end of support for OSP 16.1 

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.12

Issue: [OSDOCS-5027](https://issues.redhat.com//browse/OSDOCS-5027)

Link to docs preview: https://55324--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-notable-technical-changes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by eurijon** [comment]
lgtm

**Comment by sjstout** [comment]
LGTM!

**Comment by dulek** [comment]
I think this could go forward now?

**Comment by maxwelldb** [comment]
And @sjstout do we have any boilerplate language to use in this sort of situation if [what Eric offered](https://issues.redhat.com/browse/OSDOCS-5027?focusedId=21850187&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21850187) isn't quite right?

**Comment by sjstout** [comment]
If the issue is with the suggestion that we are not supporting 16.1 because it is in maintenance mode, how about the following rewording:
{product-title} no longer supports {rh-openstack} 16.1   as a deployment target. See  [OpenShift Container Platform on Red Hat OpenStack Platform Support Matrix](https://access.redhat.com/articles/4679401) for complete details. 

I am not aware of any boilerplate language here. 

@maxwelldb

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -1426,6 +1426,12 @@ In {product-title} v4.12, the LVM Operator has been renamed _Logical Volume Mana
 Now, you install it as a standalone Operator from the OpenShift Operator catalog.
 Logical Volume Manager Storage provides dynamic provisioning of block storage on a single, limited resources {sno} cluster.
 
+[discrete]
+[id="ocp-4-12-openstack-platform-support"]
+=== End of support for {rh-openstack} 16.1
+
+Due to {rh-openstack} 16.1 entering the maintenance phase of its lifecycle, {product-title} no longer supports that version of the platform as a deployment target.
```
[the remark this answers is not in the document]
Comment (dulek) [reply release_notes/ocp-4-12-release-notes.adoc:1433]: No objections, let's merge this!

[the remark this answers is not in the document]
Comment (maxwelldb) [reply release_notes/ocp-4-12-release-notes.adoc:1433]: @dulek Not so fast! Now begins the collection of acks on the chosen statement. And then there's a waiting period. 




And then the merge. :snail:

[the remark this answers is not in the document]
Comment (dulek) [reply release_notes/ocp-4-12-release-notes.adoc]: Well, I meant "let's proceed with this!" then. ;)

**Comment by kalexand-rh** [comment]
I agree with Stephanie. LGTM.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -1426,6 +1426,12 @@ In {product-title} v4.12, the LVM Operator has been renamed _Logical Volume Mana
 Now, you install it as a standalone Operator from the OpenShift Operator catalog.
 Logical Volume Manager Storage provides dynamic provisioning of block storage on a single, limited resources {sno} cluster.
 
+[discrete]
+[id="ocp-4-12-openstack-platform-support"]
+=== End of support for {rh-openstack} 16.1
+
+{product-title} no longer supports {rh-openstack} 16.1 as a deployment target. See [OpenShift Container Platform on Red Hat OpenStack Platform Support Matrix](https://access.redhat.com/articles/4679401) for complete details.
```
Comment (sferich888) [inline release_notes/ocp-4-12-release-notes.adoc:1433]: I don't know that I like having this announce  mid 4.12's lifecycle (especially with it being a EUS release with a 24 month lifecycle); why wasn't this part the GA announcement?

Comment (maxwelldb) [reply release_notes/ocp-4-12-release-notes.adoc:1433]: @sferich888 The request for this note came in well after release notes finalization for 4.12 GA. Not sure if Gil or @dulek can provide more background on it.

Comment (dulek) [reply release_notes/ocp-4-12-release-notes.adoc:1433]: This was hashed out by PM only by late January, we're just trying to correct a honest mistake of not having that decided before RNs were finalized. However the matrix [1] shows the correct version numbers for a while.

[1] https://access.redhat.com/articles/4679401

**Comment by dulek** [comment]
/remove-lifecycle stale

@maxwelldb, seems like this slipped through the cracks.

**Comment by maxwelldb** [comment]
@dulek Change management acks requested on Jira ticket.

**Comment by mburke5678** [comment]
@maxwelldb Please update the link to the standard format, for consistency purposes. 

`link:https://access.redhat.com/articles/4679401[OpenShift Container Platform on Red Hat OpenStack Platform Support Matrix]
`
Also. the preview link is not working. Can you update the link?

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (maxwelldb) [inline release_notes/ocp-4-12-release-notes.adoc:1433]:
```suggestion
{product-title} no longer supports {rh-openstack} 16.1 as a deployment target. See link:[https://access.redhat.com/articles/4679401[OpenShift](https://access.redhat.com/articles/4679401%5BOpenShift) Container Platform on Red Hat OpenStack Platform Support Matrix] for complete details.
```

**Comment by maxwelldb** [comment]
Done.

**Review by mandre — COMMENTED** [review]
~~LGTM~~

**edit**: actually there are more changes needed in order to remove traces of OSP 16.1 in the docs.

```
~/dev/openshift/openshift-docs enterprise-4.12
❯ ag "[ /]16\.1"
release_notes/ocp-4-12-release-notes.adoc
1378:You can now specify link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/users_and_identity_management_guide/application_credentials[application credentials] in the `clouds.yaml` files of clusters that run on {rh-openstack-first}. Application credentials are an alternative to embedding user account details in configuration files. As an example, see the following section of a `clouds.yaml` file that includes user account details:
3065:* Due to an unresolved metadata API issue, you cannot install clusters that use bare-metal workers on {rh-openstack} 16.1. Clusters on {rh-openstack} 16.2 are not impacted by this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[*BZ#2033953*])

modules/nw-osp-loadbalancer-etp-local.adoc
12:In {rh-openstack} 16.1 and 16.2, the OVN Octavia provider does not support health monitors. Therefore, setting the ETP to local is unsupported.
14:In {rh-openstack} 16.1 and 16.2, the Amphora Octavia provider does not support HTTP monitors on UDP pools. As a result, UDP load balancer services have `UDP-CONNECT` monitors created instead. Due to implementation details, this configuration only functions properly with the OVN-Kubernetes CNI plugin. When the OpenShift SDN CNI plugin is used, the UDP services alive nodes are detected unreliably.

modules/installation-osp-setting-cloud-provider-options.adoc
52:<4> This property controls whether the cloud provider creates health monitors for Octavia load balancers. Set the value to `True` to create health monitors. As of {rh-openstack} 16.1 and 16.2, this feature is only available for the Amphora provider.
65:You must set the value of the `create-monitor` property to `True` if you use services that have the value of the `.spec.externalTrafficPolicy` property set to `Local`. The OVN Octavia provider in {rh-openstack} 16.1 and 16.2 does not support health monitors. Therefore, services that have `ETP` parameter values set to `Local` might not respond when the `lb-provider` value is set to `"ovn"`.

modules/networking-osp-preparing-for-sr-iov.adoc
4:Before you use link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html-single/network_functions_virtualization_planning_and_configuration_guide/index#assembly_sriov_parameters[single root I/O virtualization (SR-IOV)] on a cluster that runs on {rh-openstack-first}, make the {rh-openstack} metadata service mountable as a drive and enable the No-IOMMU Operator for the virtual function I/O (VFIO) driver.

modules/installation-osp-provider-networks.adoc
28:You can learn more about provider and tenant networks in link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/networking-overview_rhosp-network#tenant-provider-networks_network-overview[the {rh-openstack} documentation].

modules/installation-osp-control-compute-machines.adoc
50:Additionally, for clusters that use single-root input/output virtualization (SR-IOV), {rh-openstack} compute nodes require a flavor that supports link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/configuring_the_compute_service_for_instance_creation/assembly_configuring-compute-nodes-for-performance_compute-performance#proc_configuring-huge-pages-on-compute-nodes_compute-performance[huge pages].
60:* For more information about configuring performant {rh-openstack} compute nodes, see link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html-single/configuring_the_compute_service_for_instance_creation/configuring-compute-nodes-for-performance#configuring-compute-nodes-for-performance[Configuring Compute nodes for performance].

modules/nw-ingress-controller-configuration-parameters.adoc
73:On {rh-openstack-first}, the `LoadBalancerService` endpoint publishing strategy is only supported if a cloud provider is configured to create health monitors. For {rh-openstack} 16.1 and 16.2, this strategy is only possible if you use the Amphora Octavia provider.

modules/installation-osp-configuring-sr-iov.adoc
10:If your {rh-openstack-first} deployment supports link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html-single/network_functions_virtualization_planning_and_configuration_guide/index#assembly_sriov_parameters[single root I/O virtualization (SR-IOV)], you can provision SR-IOV networks that compute machines run on.

modules/installation-osp-deploying-bare-metal-machines.adoc
29:* The {rh-openstack} link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/index[Bare Metal service (Ironic)] is enabled and accessible via the {rh-openstack} Compute API.
31:* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/configuring-the-bare-metal-provisioning-service-after-deployment#creating-the-bare-metal-flavor_bare-metal-post-deployment[a {rh-openstack} flavor].

modules/machineset-osp-adding-bare-metal.adoc
18:* The {rh-openstack} link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/index[Bare Metal service (Ironic)] is enabled and accessible by using the {rh-openstack} Compute API.
20:* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/sect-configure#creating_the_bare_metal_flavor[an {rh-openstack} flavor].

modules/installation-osp-provider-network-preparation.adoc
13:* The link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/networking-overview_rhosp-network#install-networking_network-overview[{rh-openstack} networking service (Neutron) is enabled] and accessible through the {rh-openstack} networking API.
14:* The {rh-openstack} networking service has the link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/config-allowed-address-pairs_rhosp-network#overview-allow-addr-pairs_config-allowed-address-pairs[port security and allowed address pairs extensions enabled].
37:To learn more about creating networks on {rh-openstack}, read link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/networking-overview_rhosp-network#tenant-provider-networks_network-overview[the provider networks documentation].
56:* Optional: To secure the network, create link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/config-rbac-policies_rhosp-network#proc_create-rbac-policies_config-rbac-policies[role-based access control (RBAC)] rules that limit network access to a single project.

modules/installation-openstack-sr-iov-requirements.adoc
23:** For more information about configuring performant {rh-openstack} compute nodes, see link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html-single/configuring_the_compute_service_for_instance_creation/configuring-compute-nodes-for-performance#configuring-compute-nodes-for-performance[Configuring Compute nodes for performance].

modules/supported-platforms-for-openshift-clusters.adoc
16:* {rh-openstack-first} versions 16.1 and 16.2
42:* {rh-openstack} versions 16.1 and 16.2

modules/installation-osp-setting-worker-affinity.adoc
21:You can learn more about link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/configuring_the_compute_service_for_instance_creation/assembly_configuring-instance-scheduling-and-placement_scheduling-and-placement[{rh-openstack} instance scheduling and placement] in the {rh-openstack} documentation.
```

**Comment by xltian** [comment]
lgtm

**Pull request merged**